recipes/jabber

```diff
@@ -1,4 +1,5 @@
 (jabber
- :repo "legoscia/emacs-jabber"
- :fetcher github
- :files ("*.el" "*.texi" ("jabber-fallback-lib" "jabber-fallback-lib/hexrgb.el")))
+ :fetcher git
+ :url     "https://codeberg.org/emacs-jabber/emacs-jabber.git"
+ :branch  "production"
+ :files   ("*.org" "*.el" "*.texi" ("jabber-fallback-lib" "jabber-fallback-lib/hexrgb.el")))
```

Comment:
I think `:defaults` will be useful here because it will exclude .dir-locals.el and automatically include the *.el and *.texi files you want to include.
```suggestion
 :fetcher git
 :url     "https://codeberg.org/emacs-jabber/emacs-jabber.git"
 :branch  "production"
 :files   (:defaults "*.org" ("jabber-fallback-lib" "jabber-fallback-lib/hexrgb.el")))
```
You may also want to exclude some of the .org files (do users really need jabber.org?)

Reply:
Since I *just* finished making cosmetic changes to a bunch of recipes (@riscy what do you think about #8072), a cosmetic note:

While I am an aligner myself, I think we shouldn't align values in recipes, mostly because that is rarely done, i.e., for consistency.

Reply:
I agree with that - beyond it being rare, sometimes the recipes get batch processed with tools that might not have the same opinions about formatting, so I wouldn't bother.  In practice I wouldn't personally consider this a blocker, though (if we wanted it to be a blocker, I think enforcement would be pretty easy via CI)